### ISSUE TITLE
Buttons loading as list 

### ISSUE THREAD

**Issue by JasmeetVirdi** [Microsoft/BotFramework-WebChat#1566]
I am using a thumbnail card to display a title, subtitle and buttons. The buttons appear fine in emulator. However using web chat on server, the buttons are appearing as bullet list, i.e. _thumbnail is not loading correctly in web chat_.

**Comment by compulim** [comment]
This sounds familiar and I remember there is an issue (or by design) on Bot Builder SDK side.

@corinagum can you help and look it up? @stevengum might know more about it.

**Comment by corinagum** [comment]
@JasmeetVirdi could you share your thumbnail card for reproduction purposes? Thank you!

**Comment by stevengum** [comment]
@JasmeetVirdi, a few questions... 

1. Which version of webchat are you using? Which version of the emulator?
2. Which version of the SDK are you using?

Also, could you provide a screenshot of the behavior in webchat? Thanks!

**Comment by corinagum** [comment]
@compulim - I rediscovered our related issue: https://github.com/Microsoft/BotFramework-WebChat/issues/1333

**Comment by JasmeetVirdi** [comment]
Hello @stevengum ,apologies for late reply.
I am using SDK version 3
The version of emulator is 3.5.36
i am not really sure abt the webchat version.
Also, @corinagum the related issue's overall conclusion is not clear.Will need some help regarding that.

**Comment by corinagum** [comment]
@JasmeetVirdi oops sorry, I should have been more clear - I was pointing out that we have had this issue reported before. Getting your Web Chat version will be a big help for us.

As for the version of Web Chat, are you using `iframe` on your page? If so, your Web Chat is v3. If you are using NPM or CDN, please tell me the version or package number and we can figure out from there. :)

**Comment by JasmeetVirdi** [comment]
Hi @corinagum sorry for late reply,
i am using iframe on my page so that will be Web Chat v3.

**Comment by JasmeetVirdi** [comment]
@corinagum  I found that,oauth signin card is also not working as desired,so probably an issue with the cards.
PFB the screenshot
![image](https://user-images.githubusercontent.com/13176977/51883043-44113600-23a7-11e9-9bdb-baec53ab5c35.png)

**Comment by corinagum** [comment]
Hi @JasmeetVirdi, I'm sorry about the delay in getting back to you!
I looked into this issue, and unfortunately I'm not able to repro it in v3. I used the thumbnail code you provided me and got it to display as expected in v3
![image](https://user-images.githubusercontent.com/14900841/52749435-05ca6800-2f9e-11e9-8333-0975affc3d5a.png)

The only thing I can think of that may be causing this issue is that perhaps you haven't set your reply to have an attachment layout of type List? 

```
reply.AttachmentLayout = AttachmentLayoutTypes.List;
reply.Attachments = new List<Attachment>();
reply.Attachments.Add(cardAttachment);
await context.PostAsync(reply);
```

Since this appears to be a bot implementation issue, I recommend taking this to [Stack Overflow](https://stackoverflow.com/tags/botframework) if you are still having problems. Thanks again for your patience!
